editor/editor_build_profile.cpp

```diff
@@ -40,34 +41,58 @@
 #include "editor/themes/editor_scale.h"
 #include "scene/gui/separator.h"
 
+#include "modules/modules_enabled.gen.h" // For mono.
+
 const char *EditorBuildProfile::build_option_identifiers[BUILD_OPTION_MAX] = {
 	// This maps to SCons build options.
 	"disable_3d",
-	"disable_2d_physics",
-	"disable_3d_physics",
-	"disable_navigation",
+	"disable_navigation_2d",
+	"disable_navigation_3d",
 	"disable_xr",
-	"rendering_device", // FIXME: there's no scons option to disable rendering device
-	"opengl3",
+	"module_openxr_enabled",
+	"wayland",
+	"x11",
+	"rendering_device", // FIXME: There's no scons option to disable rendering device.
+	"forward_plus_renderer",
+	"forward_mobile_renderer",
 	"vulkan",
+	"d3d12",
+	"metal",
+	"opengl3",
+	"disable_physics_2d",
+	"module_godot_physics_2d_enabled",
+	"disable_physics_3d",
+	"module_godot_physics_3d_enabled",
+	"module_jolt_physics_enabled",
 	"module_text_server_fb_enabled",
 	"module_text_server_adv_enabled",
 	"module_freetype_enabled",
 	"brotli",
 	"graphite",
-	"module_msdfgen_enabled"
+	"module_msdfgen_enabled",
 };
 
 const bool EditorBuildProfile::build_option_disabled_by_default[BUILD_OPTION_MAX] = {
 	// This maps to SCons build options.
 	false, // 3D
-	false, // PHYSICS_2D
-	false, // PHYSICS_3D
-	false, // NAVIGATION
+	false, // NAVIGATION 2D
+	false, // NAVIGATION 3D
```

Comment:
This is meant to be the suffix after `BUILD_OPTION_` so it should use underscores (same for other occurrences).

```suggestion
	false, // NAVIGATION_2D
	false, // NAVIGATION_3D
```